backend/server/mongodb/models/TrainingLog.ts

```diff
@@ -0,0 +1,45 @@
+import mongoose from "mongoose";
+import { TrainingLog, ServiceAnimalBehavior } from "src/utils/types";
+
+const ServiceAnimalBehaviorSchema = new mongoose.Schema<ServiceAnimalBehavior>({
+  description: String,
+  repeat: Number,
+});
+
+const TrainingLogSchema = new mongoose.Schema<TrainingLog>({
+  date: {
+    type: Date,
+    required: true,
+    index: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  skills: {
+    type: [String],
+    required: true,
+  },
+  trainingHours: {
+    type: Number,
+    required: true,
+  },
+  behavior: {
+    type: ServiceAnimalBehaviorSchema,
+    required: true,
+  },
+  animal: {
+    // TODO: replace with the actual schema
+    type: Object,
+    required: true,
+  },
+  video: {
```

Comment:
I think I can agree with keeping this optional and omitted -- tbh it doesn't really matter but if we want to keep things organized with our DB, then omitting it seems smart.